src/Tool/Wrapper/EntityWrapper.php

```diff
@@ -81,34 +82,32 @@ public function getRootObjectName()
     /**
      * {@inheritdoc}
      */
-    public function getIdentifier($single = true)
+    public function getIdentifier($single = true, $flatten = false)
```

Comment:
IMO, adding an argument to a method that can be overridden by the user is a BC break.

Reply:
Ah yes. Forgot about that. You are right, I'd also see that as a BC break.
I could alleviate this by leaving the argument list as is and using `func_get_args` for the time beeing and adding a deprecation message. Then just change the argument list when the next major is going to be released.
This approach has been used by e.g. Symfony for forward compatibility.

Would it be OK to use it here?

Reply:
Sure, we can provide a BC layer this way.

Reply:
Ich have implemented the method in the BC compatible way. I just ommited the deprecation message, as I found no soild way to have it always run.